src/client/camera.cpp

```diff
@@ -549,8 +549,13 @@ void Camera::updateViewingRange()
 	f32 viewing_range = g_settings->getFloat("viewing_range");
 	f32 near_plane = g_settings->getFloat("near_plane");
 
+#ifdef __ANDROID__
```

Comment:
might be a good idea to leave a comment here on why it's different on android (and presumingly other mobile devices?)

Reply:
Good idea. Will do.

Reply:
Added a one-line comment